src/data/ab-attrs/post-damage-ab-attr.ts

```diff
@@ -11,8 +11,8 @@ export class PostDamageAbAttr extends AbAttr {
     _damage: number,
     _passive: boolean,
     _simulated: boolean,
-    _args: any[],
     _source?: Pokemon,
+    ..._args: any[]
```

Comment:
should this be unknown like the rest of the changes or is it not needed since nothing extends it?
```suggestion
    ..._args: unknown[]
```

Reply:
+1

Reply:
I left in `args` fields for all of the different ability attribute types, even if they aren't currently used. I must've just missed this one.

Side note: I'm gonna make a follow-up PR to clean up the `applyXAbAttrs` functions (I'll probably end up merging them into one function using a method similar to #199 ). That should also help with any extra/unused `args`.